src/components/alphabet/images-space/ImagesSpace.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="images-space">
+        <div class="image-item" v-for="(uri, index) in getImagesUri()"
+            :key="index"
+            v-on:click="$emit('imageSelected', index)">
+            <img v-bind:src="uri"/>
```

Comment:
use shorthands for `v-bind` and `v-on` https://vuejs.org/v2/guide/syntax.html#v-bind-Shorthand